Branch-SDK/src/main/java/io/branch/referral/ServerRequest.java

```diff
@@ -645,11 +645,6 @@ public BRANCH_API_VERSION getBranchRemoteAPIVersion() {
         return BRANCH_API_VERSION.V1;  // Default is v1
     }
     
-    public void reportTrackingDisabledError() {
```

Comment:
public method for internal use... I just copy pasted the implementation of the method to places where the method was invoked.